Kernel/Arch/i386/Boot/boot.S

```diff
@@ -31,6 +31,11 @@ stack_bottom:
 .skip 32768
 stack_top:
 
+.section .pagetable
```

Comment:
"page" and "table" is two words, please separate them by an underscore.
Also, there is more than one table, so let's say "tables" :)
```suggestion
.section .page_tables